lib/graphql/schema.flow.js

```diff
@@ -1764,6 +1819,7 @@ export type UserMetadata = {|
   marketingConsentAccepted: $ElementType<Scalars, 'Boolean'>,
   phoneNumberVerificationRequired: $ElementType<Scalars, 'Boolean'>,
   signupCompleted: $ElementType<Scalars, 'Boolean'>,
+  categorizationScreenShown?: ?$ElementType<Scalars, 'Boolean'>,
```

Comment:
🙃 Not part of this PR, should be updated here: https://github.com/kontist/js-sdk/pull/111. Will ask @p-janik if it can be merged.

Reply:
Update: Merged, can you please update your branch]?

Reply:
This was added automatically when running `schema:generate` - I guess I could cherry-pick only the Asset-related parts?

Reply:
Yes, I know, but now you can just update your branch with develop, it was already added there with #111.

Reply:
Done.